ros/src/computing/perception/localization/packages/ekf_localizer/include/ekf_localizer/ekf_localizer.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2018-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <iostream>
+#include <vector>
+#include <chrono>
+
+#include <ros/ros.h>
+#include <geometry_msgs/PoseArray.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/PoseWithCovarianceStamped.h>
+#include <geometry_msgs/TwistStamped.h>
+#include <geometry_msgs/TransformStamped.h>
+#include <std_msgs/Float64.h>
+#include <std_msgs/Float64MultiArray.h>
+#include <tf2/utils.h>
+#include <tf2/LinearMath/Quaternion.h>
+#include <tf2_ros/transform_listener.h>
+#include <tf2_ros/transform_broadcaster.h>
+
+#include <autoware_msgs/VehicleStatus.h>
+
+#include "amathutils_lib/kalman_filter.hpp"
+#include "amathutils_lib/kalman_filter_delayed_measurement.hpp"
+
+class EKFLocalizer
+{
+public:
+  EKFLocalizer();
+  ~EKFLocalizer();
+
+private:
+  ros::NodeHandle nh_, pnh_;
+  ros::Publisher pub_pose_, pub_pose_cov_, pub_twist_, pub_debug_, pub_measured_pose_, pub_yaw_bias_;
+  ros::Subscriber sub_initialpose_, sub_pose_, sub_vehicle_status_, sub_twist_, sub_pose_with_cov_;
+  ros::Timer timer_control_, timer_tf_;
+  tf2_ros::TransformBroadcaster tf_br_;
+
+  KalmanFilterDelayedMeasurement ekf_;
+
+  /* parameters */
```

Comment:
please use doxygen comment [formatting](http://www.doxygen.nl/manual/docblocks.html#cppblock)

Reply:
update to doxygen format. pls check it.